resources/assets/lib/beatmapsets/beatmapset-json.ts

```diff
@@ -93,6 +93,6 @@ export interface BeatmapsetJson {
   nsfw: boolean;
   status: BeatmapsetStatus;
   title: string;
-  title_unicode: string | null;
+  title_unicode: string;
```

Comment:
should update `artist_unicode` one as well?

Reply:
huh, I thought I updated it 🤔